sros2/package.xml

```diff
@@ -16,11 +16,13 @@
   <exec_depend>ament_index_python</exec_depend>
   <exec_depend>python3-lxml</exec_depend>
   <exec_depend>python3-cryptography</exec_depend>
+  <exec_depend>python3-importlib-resources</exec_depend>
 
   <test_depend>ament_copyright</test_depend>
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_mypy</test_depend>
   <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-importlib-resources</test_depend>
```

Comment:
```suggestion
```
From the [docs](http://wiki.ros.org/catkin/package.xml#Build.2C_Run.2C_and_Test_Dependencies) "Test Dependencies specify only additional dependencies for unit tests. They should never duplicate any dependencies already mentioned as build or run dependencies."

Reply:
I went ahead and merged this suggestion. I'll let another @ros2/security-reviewers give the second approval